internal/store/store.go

```diff
@@ -247,7 +247,7 @@ func NewPrivateUnauth(c Config) (*Store, error) {
 	s.baseURL = atlas.CloudURL
 
 	if configURL := c.OpsManagerURL(); configURL != "" {
-		s.baseURL = s.apiPath(configURL)
+		s.baseURL = configURL
```

Comment:
drive-by fix

Reply:
[q] are sure? this is used to create the owner and I'm pretty sure it was working as expected, remember that ops manager doesn't set up the `/api` segment as part of the base url

Reply:
also this is used for the default cluster version, I'll add cluster deployment tests to see if this works

Reply:
actually this may be right, I think this is only used for the default version and I think that one always uses teh default setting because of reasons